wp-content/plugins/core/src/Integrations/Site_Brand/Site_Brand_Subscriber.php

```diff
@@ -0,0 +1,34 @@
+<?php
+declare( strict_types=1 );
+
+namespace Tribe\Project\Integrations\Site_Brand;
```

Comment:
What plugin is this integrating with? Just as in the theme, the intention of the "Integrations" namespace is to contain code that integrates our project with 3rd-party plugins.

Reply:
Good point...I wondered about this. Perhaps this belongs in the `Admin` directory or a new `Branding` directory, @jbrinley?

Reply:
I like `Branding`, either top-level or under `Theme`.